Source/DafnyDriver/BoogieXmlConvertor.cs

```diff
@@ -54,6 +55,13 @@ public static class BoogieXmlConvertor {
         } else if (loggerName == "csv") {
           var csvLogger = new CSVTestLogger();
           csvLogger.Initialize(events, parameters);
+        } else if (loggerName == "text") {
+          // This doesn't actually use the XML converter. It instead uses a collection of VerificationResult
+          // objects. Ultimately, the other loggers should be converted to use those objects, as well,
+          // and then it would make sense to rename this class.
```

Comment:
As in making it `TextTestLogger` instead?

Reply:
Ah, I meant the class containing this code: `BoogieXmlConverter` will probably no longer be appropriate. ;)

Reply:
Oh yeah that class can be fully deleted once we switch over, and all related code.